APItokenmanager/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Assuming you already have a Dockerfile setup for Django
+FROM python:3.8
+
+# Set the working directory
+WORKDIR /app
+
+# Copy project code
+COPY . /app
+
+# Install dependencies
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
```

Comment:
Combine into single clause to limit layers in your docker image

Reply:
Combined into single clause as suggested.